CHANGELOG.md

```diff
@@ -3,6 +3,20 @@
 Documentation for rocALUTION is available at
 [https://rocm.docs.amd.com/projects/rocALUTION/en/latest/](https://rocm.docs.amd.com/projects/rocALUTION/en/latest/).
 
+## rocALUTION 3.2.1 for ROCm 6.3.0
+
+### Additions
+* Support for gfx1200, gfx1201 and gfx1151.
+
+### Changes
+
+* Change default compiler from hipcc to amdclang in install script and cmake files.
```

Comment:
Changed the default compiler in the install script and in the cmake files from `hipcc` to `amdclang`.